DirectEmployers/MyJobs
# Fix get activities behavior

## What's this PR do?
Changes user.get_activities(company) to only return activities that the company provided has app access to.
## Screenshots (if appropriate)

## Where should the reviewer start?
Anywhere user.get_activities is used

## How should this be manually tested?
Have more than one role for a couple companies
Have those companies have different app access levels
Call user.get_activities and provide each company
Notice the difference in activities returned

## Any background context you want to provide?
This is currently a guidance requested PR. Most of the activity-related functions that use this function also do a separate call to make sure that the activities returned also belong to the company provided. This is not the case in the Javascript User.Can, however, and it made me wonder why we would return activities in get_activities that don't belong to the company provided. While we -could- change the javascript to filter out activities that don't relate to the company provided.. Is there any reason to provide these activities in the first place?
